api/rtx/src/uvisor_allocator.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "uvisor_allocator.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "uvisor-lib/uvisor-lib.h"
```

Comment:
Could you include `uvisor-lib.h` first? I'd say the same for the `rt_*.h` files but if they are not self contained leave them here.